decidim-comments/app/cells/decidim/comments/comment_form_cell.rb

```diff
@@ -17,6 +17,14 @@ def comment_as_for(form)
 
       private
 
+      def cache_hash
+        hash = []
+        hash.push(I18n.locale)
+        hash.push(model.cache_key)
+        hash.push(current_user.try(:id))
```

Comment:
Adding the current_user.id to every cache_hash in this cell seems like it'll grow a lot, although as far as I see it's necessary for the "comment as" dropdown. Does it make sense having this cell cached?

Reply:
it does make to have it cached, as for each one the comments there is a reply box, that is being rendered as hidden, which adds to render time.

Reply:
👍🏽